include/config/libinput.h

```diff
@@ -21,6 +21,8 @@ struct libinput_category {
 	int left_handed;
 	enum libinput_config_tap_state tap;
 	enum libinput_config_tap_button_map tap_button_map;
+	enum libinput_config_drag_state tap_and_drag;
```

Comment:
The less-than-zero check with an enum is a great catch.

I suggest that we change these enums to 'int' instead though, rather than doing an `(int)` cast later.

Reply:
I agree. I fixed my code.